bitmap-ram-rod/test/transformer-test.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
```

Comment:
Good tests overall. The one thing you really aren't doing much of is testing the entirety of the write process. You should be making your you can appropriately write new bmp files and cleaning them out when you are done.